golint does not print backtrace

It is a pain to debug errors like
```
  Error: S1002: should omit comparison to bool constant, can be simplified to `connected` (gosimple)
  Error: SA5001: should check returned error before deferring conn.Close() (staticcheck)
  Error: SA5001: should check returned error before deferring conn.Close() (staticcheck)
```
without having any information about what file and line they are referred to.

Yes, we need to fix it on CI, but on the local machine you could to use the make target:

```
$ make golangci-lint 
golangci-lint run -E goimports -D errcheck
connection_pool/connection_pool.go:46:19: undeclared name: `tarantool` (typecheck)
	Discovered(conn *tarantool.Connection, role Role) error
```

> Yes, we need to fix it on CI ( [golangci/golangci-lint-action#119](https://github.com/golangci/golangci-lint-action/issues/119) , [tarantool/tt#116](https://github.com/tarantool/tt/pull/116) ), but on the local machine you could to use the make target:
> 
> ```
> $ make golangci-lint 
> golangci-lint run -E goimports -D errcheck
> connection_pool/connection_pool.go:46:19: undeclared name: `tarantool` (typecheck)
> 	Discovered(conn *tarantool.Connection, role Role) error
> ```

Either something had changed between versions or flags are different from the CI ones.

![image](https://user-images.githubusercontent.com/20455996/204274633-0e2cbb47-9fe9-4d05-9597-e432de1cfc53.png)

--- a/.github/workflows/check.yaml
+++ b/.github/workflows/check.yaml
@@ -37,7 +37,8 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v2
+        uses: golangci/golangci-lint-action@v3
+        continue-on-error: true
         with:
           # The suppression of the rule `errcheck` may be removed after adding
           # errors check in all methods calling EncodeXxx inside.
@@ -47,7 +48,16 @@ jobs:
           # The `//nolint` workaround was not the acceptable way of warnings suppression,
           # cause those comments get rendered in documentation by godoc.
           # See https://github.com/tarantool/go-tarantool/pull/160#discussion_r858608221
+          #
+          # The first run is for GitHub Actions error format.
           args: -E goimports -D errcheck
+
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v3
+        with:
+          # The second run is for human-readable error format with a file name
+          # and a line number.
+          args: --out-${NO_FUTURE}format colored-line-number -E goimports -D errcheck
 
   codespell:
     runs-on: ubuntu-latest

